src/pages/programs/program-observation-submission/program-observation-submission.html

```diff
@@ -1,14 +1,8 @@
 <ion-header>
```

Comment:
This page is changed every where for submissionId and submissionStatus to _id and status